CHANGELOG.md

```diff
@@ -29,6 +29,12 @@
 - [DOCS] Added design component documentation for cards, buttons, images, bylines, timestamps, footer, sidebar
 - [DOCS] Added example "stories" for buttons
 
+## `2.10.4`
+- [CHANGE] added newsletter signup component for article page
+- [CHANGE] added link for terms on newsletter signup components
+- [CHANGE] added GA tracking to newsletter signup components 
+- [CHANGE] added location field for mailchimp tracking to newsletter signup components 
+
```

Comment:
I think merging these into master branch should have part of a separate PR but if they're in here now i guess we can keep them